tools/integration_tests/implicitdir/implicitdir_test.go

```diff
@@ -160,26 +187,15 @@ func TestMain(m *testing.M) {
 		os.Exit(1)
 	}
 
-	if err := mountGcsfuse(); err != nil {
-		log.Printf("mountGcsfuse: %v\n", err)
-		os.Exit(1)
-	}
+	flags := []string{"--experimental-enable-storage-client-library=true",
+		"--experimental-enable-storage-client-library=false",
+		"--implicit-dirs=true",
+		"--implicit-dirs=false"}
 
-	log.Printf("Test log: %s\n", logFile)
-
-	// Creating a temporary directory to store files
-	// to be used for testing.
-	var err error
-	tmpDir, err = os.MkdirTemp(mntDir, "tmpDir")
+	successCode, err := executeTest(flags, m)
 	if err != nil {
-		logAndExit(fmt.Sprintf("Mkdir at %q: %v", mntDir, err))
+		os.Exit(1)
```

Comment:
you are not logging the error. do we get in the logs why it failed?

Reply:
fixed